### PR TITLE
Igal/single node v2 on eran

### PR DESCRIPTION
**Pull request by tsorya**
/hold

### CODE REVIEW

**Comment by cfergeau** [comment]
What is this about? Does this need early reviews?

**Comment by gbraad** [comment]
Hi, can you explain what this PR tries to solve and what this aligns with? There is an effort to create a Single Node Cluster at https://github.com/code-ready/snc/ which is used by CRC. Also, for resource tuning, effort is being made with the Single Node Profile enhancement, and is work on by the same team. Would appreciate some back story or more info

File: pkg/asset/tls/tls.go [hunk 1]
```diff
@@ -24,6 +24,9 @@ const (
 	// ValidityOneDay sets the validity of a cert to 24 hours.
 	ValidityOneDay = time.Hour * 24
 
+	// ValidityOneMinute sets the validity of a cert to 1 month.
+	ValidityOneMonth = ValidityOneDay * 30
```
Comment (gbraad) [inline pkg/asset/tls/tls.go:28]: Same comment as here: https://github.com/openshift/installer/pull/4308#discussion_r530360087

This is a no-go situation. Initial certificates are deemed to be only 24hrs, after which they rotate to the expected 30 days validity.

**Comment by staebler** [comment]
This seems to be obsolete. Feel free to re-open if I am mistaken.
/close

**Pull request closed without merging**